Sources/SwiftProtobuf/ExtensionFields.swift

```diff
@@ -39,6 +39,12 @@ public protocol AnyExtensionField: CustomDebugStringConvertible {
   var isInitialized: Bool { get }
 }
 
+#if DEBUG
+public protocol _AnyExtensionField: CustomDebugStringConvertible {}
+#else
+public protocol _AnyExtensionField {}
+#endif
```

Comment:
Not sure if this adds any extra overhead. I couldn't find another good way to conditionally have protocol conformance.

Reply:
Right, that should work fine.

Reply:
@allevato made this change